### ISSUE TITLE
Cluster name isn't populated for all events. 

### ISSUE THREAD

**Issue by benarent** [gravitational/teleport#5856]
### Description

**What happened**:

While reviewing .. I noticed that "Session Data", "SSO Login" and "User Updated" are missing `cluster_name`

```
{
  "attributes": {
    "coderwall-charity": [
      "honoraries"
    ],
    "gravitational": [
      "devc",
      "gravitational-employees",
      "wikireaders",
      "web-developers",
      "officessh",
      "wikiwrite"
    ]
  },
  "cluster_name": "",
  "code": "T1001I",
  "ei": 0,
  "event": "user.login",
  "method": "github",
  "success": true,
  "time": "2021-03-04T19:26:38.275Z",
  "uid": "d520bd8d-6929-45ea-bf4f-d011bb5b6c6e",
  "user": "benarent"
}
```

**What you expected to happen**:
Cluster name should be shown 

### Reproduction Steps
This was tested with `6.0.0-rc.1`

**Comment by benarent** [comment]
cc @a-palchikov

**Comment by zmb3** [comment]
This was likely fixed with #5916.

I checked all 3 events on teleport 10 and all have the cluster name set.
